tests/service/data/TestAnalyzer/TestAnalyzer.fsproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
a key part of the roslyn analyzer business is they are netstandard only. it's worth requiring here too

Reply:
sure yup makes sense